metaflow/plugins/argo/argo_workflows.py

```diff
@@ -2397,11 +2397,12 @@ def _pager_duty_notification_links(self):
         if ARGO_WORKFLOWS_UI_URL:
             links.append(
                 {
-                    "href": "%s/workflows/%s/%s"
-                    % (
-                        ARGO_WORKFLOWS_UI_URL.rstrip("/"),
-                        "{{workflow.namespace}}",
-                        "{{workflow.name}}",
+                    "href": ARGO_WORKFLOWS_UI_URL.rstrip("/").format(
```

Comment:
ARGO_WORKFLOWS_UI_URL should be set as:

`https://metaflow.foo.argo.com/{uri}/{namespace}/{workflow_name}`